### PR TITLE
Upgrade Hedley to v15

### PR DESCRIPTION
**Pull request by Phytolizer**

### CODE REVIEW

**Comment by Phytolizer** [comment]
first time contributing to this, so lmk if i missed something :)

File: subprojects/hedley.wrap [hunk 1]
```diff
@@ -1,7 +1,7 @@
 [wrap-file]
-directory = hedley-11
-source_url = https://codeload.github.com/nemequ/hedley/tar.gz/v11
-source_filename = hedley-11.tar.gz
-source_hash = 7a8cc16a66afc1ac22c4b99b7b798fc4326d217dd4f97facb43181e84fa82f88
+directory = hedley-15
+source_url = https://codeload.github.com/nemequ/hedley/tar.gz/v15
+source_filename = hedley-15.tar.gz
+source_hash = e91c71b58f59d08c7b8289be8f687866863d934dfaa528e4be30b178139ae863
 patch_directory = hedley
```
Comment (neheb) [inline subprojects/hedley.wrap:7]: I'd remove this blank line.

**Comment by neheb** [comment]
Need to update releases.json

File: subprojects/hedley.wrap (hunk 1 above)
Comment (Phytolizer) [reply subprojects/hedley.wrap:7]: Sure thing.

**Comment by Phytolizer** [comment]
> Need to update releases.json

Thought I did this in the 2nd-3rd commit?

**Comment by neheb** [comment]
Lol other way around. See the others.

**Comment by Phytolizer** [comment]
Apologies, still learning!

**Comment by Phytolizer** [comment]
Feel free to squash this once it works, I feel silly :)

**Comment by Phytolizer** [comment]
How did this pass checks before? I'm pretty sure there wasn't a provide section originally.

**Comment by neheb** [comment]
New CI thing.

**Comment by Phytolizer** [comment]
Ok, I decided to stop throwing stuff at the wall and run the sanity_checks.py locally. Hope it works now.

**Pull request merged**